src/tests/integration_test/wallet_test.cljs

```diff
@@ -0,0 +1,63 @@
+(ns tests.integration-test.wallet-test
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    legacy.status-im.events
+    [legacy.status-im.multiaccounts.logout.core :as logout]
+    legacy.status-im.subs.root
+    status-im.events
+    status-im.navigation.core
+    status-im.subs.root
+    [test-helpers.integration :as h]
+    [tests.integration-test.constants :as test-constants]
+    [utils.re-frame :as rf]
+    [utils.security.core :as security]))
+
+(defn get-default-account
+  [accounts]
+  (first (filter :wallet accounts)))
+
+(def send-amount "0.0001")
+
+(deftest wallet-send-test
```

Comment:
this is one of the main focuses of this pr. 🦅

Reply:
It seems good @J-Son89 


I'd like to have a clearer way to read it, but it seems the re-frame's macro `wait-for` is making the reading harder.

Looking at the implementation of that macro, it returns:

```clojure
 ;; ...
(wait-for* ~ids ~failure-ids (fn [~event-sym] ~@body))
```

and `wait-for*` is a function, we could probably use it in our own macro for testing.

@ilmotta wdyt?

Reply:
yeah the wait for makes it ugly to write too.

one other feature that would be nice, often we want to wait on multiple keys and in order, but we don't support this.

we can only wait for multiple but once one is satisfied then it moves forward